infra/nightly-resources/index.js

```diff
@@ -258,12 +258,12 @@ async function loadBenchmarks(compareTo) {
 
     // Add warnings
     let warningContainer = document.getElementById("warnings");
-    // make warning red
-    warningContainer.style.color = "red";
+    
     warningContainer.innerHTML = "";
     warnings.forEach((warning) => {
-        // make warnings red
         let warningElement = document.createElement("p");
+        // make warning red with warning class
+        warningElement.classList.add("warning");
```

Comment:
Instead of this, can't you just add this in the css? I don't like manipulating things like this in the JS, it's better to just put the right selectors and stuff in the html file.
```
#warnings {
  color: red;
}
```